scripts/tfcloud/tfcloud.sh

```diff
@@ -536,9 +536,9 @@ function make_curl_request() {
     esac
   done
 
-  command="curl -sS -L -w '%{http_code}' --header 'Authorization: Bearer xxxxxx' --header 'Content-Type: application/vnd.api+json' $options -- '"${url}"' 2> >(tee /dev/stderr)"
```

Comment:
Can you explain why you are removing the redirection? Can you also show a test evidence of the confirm the code is working?